client/src/components/shell/navbar/Navbar.jsx

```diff
@@ -12,6 +12,7 @@ import NavbarLogout from "./NavbarLogout";
 const AppNavbar = ({ opened, setOpened }) => {
   const { classes } = useNavbarStyles();
   const smallScreen = useMediaQuery("(max-width: 768px)");
+  const isUser = localStorage.getItem("userType") === "user";
```

Comment:
This gets the job done, but we could use our UserContext as well, like so:
`const { userType } = useContext(UserContext)`
and then compare it below in the JSX like:
`{userType === "user" ? ...}`
But honestly we can leave this like it is.